scripts/reboot

```diff
@@ -192,6 +203,13 @@ function parse_options()
             f )
                 REBOOT_FLAGS+=" -f"
                 ;;
+            d )
+                REBOOT_DPU="yes"
+                DPU_MODULE_NAME="$OPTARG"
+                ;;
+            p )
+                PRE_SHUTDOWN="yes"
```

Comment:
@vvolam why pre-shutdown is an option and not mandatory for smartwitch?

Reply:
@prgeor It is only required on DPU and invalid on NPU.